lib/std/start.zig

```diff
@@ -307,26 +305,10 @@ fn _start() callconv(.Naked) noreturn {
             \\ andi sp, sp, -16
             \\ tail %[posixCallMainAndExit]@plt
             ,
-            .mips, .mipsel =>
+            .mips, .mipsel, .mips64, .mips64el =>
             // The lr is already zeroed on entry, as specified by the ABI.
             \\ addiu $fp, $zero, 0
             \\ move $a0, $sp
-            \\ .set push
-            \\ .set noat
-            \\ addiu $1, $zero, -16
```

Comment:
* https://github.com/rui314/psabi/blob/main/mips.pdf §3-28

No authoritative document seems to exist for 64-bit MIPS, but: https://github.com/torvalds/linux/blob/933069701c1b507825b514317d4edd5d3fd9d417/arch/mips/kernel/process.c#L708-L718